HISTORY.rst

```diff
@@ -3,7 +3,11 @@
 Release History
 ===============
 
-0.0.20 (2018-05-25)
```

Comment:
Let's keep the dates as +0.0.21 (2018-06-01) and 0.0.20 (2018-05-25). Currently I think by mistake the 25th may has been changed to 29th may because of multiple iterations.